recipes/hyperspyui/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "hyperspyUI" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c666d44e6330cd535590d00e7a11163d200b286ac99f8686aef8d94d063bf635
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  entry_points:
+    - hyperspyui = hyperspyui.__main__:main
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+
+  run:
+    - python >=3.5
+    - hyperspy-gui-traitsui
+    - pyqode.python
+    - autopep8
+    - pep8
+    - qtconsole
+    - qtpy
+    - ffmpeg
+
+test:
+  imports:
+    - hyperspyui
+    - hyperspyui.__main__
+
+  # commands:
+  #  - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'hyperspyui -h'
+  #  - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'pytest --pyargs hyperspyui'
+
+  # requires:
+  #  - pytest
+  #  - pytest-qt
+
+app:
+  entry: hyperspyui
+  icon: hyperspy128.png
+  summary:  "HyperSpyUI"
+  type: desk
+
+about:
+  home: https://github.com/hyperspy/hyperspyUI
+  license: GNU General Public v3 (GPLv3)
+  license_family: GPL3
+  summary: "HyperSpyUI tries to bring a streamlined user interface to the powerful multi-dimensional analysis capabilities of HyperSpy. HyperSpy is an open source Python library which provides tools to facilitate data analysis of multidimensional datasets.
+
+HyperSpy aims at making it easy and natural to apply analytical procedures that operate on an individual signal to multidimensional arrays, as well as providing easy access to analytical tools that exploit the multidimensionality of the dataset.
+
+While the UI tries to create a simple and intuitive interface to HyperSpy, it still retains the raw power of HyperSpy via the UI’s built in IPython console, which runs on the same Python kernel as the UI.
+"
+  license_file: COPYING.txt
```

Comment:
I am not sure, but I hope that it would pick up correctly the license in the pypi package?

Reply:
Yes, conda-build raises an exception if it fails to find the `license_file`.

Reply:
Thanks for the confirmation!